biojava-genome/src/test/java/org/biojava/nbio/genome/io/fastq/ConvertTest.java

```diff
@@ -18,25 +18,35 @@
  *      http://www.biojava.org/
  *
  */
-package org.biojava.nbio.sequencing.io.fastq;
+package org.biojava.nbio.genome.io.fastq;
 
 import java.io.File;
 import java.io.FileWriter;
 import java.util.List;
 import java.util.Map;
 
-import junit.framework.TestCase;
+import org.biojava.nbio.genome.io.fastq.Fastq;
```

Comment:
... did these all come in via an IDE?

Reply:
Somehow eclipse considered them necessary... They are indeed redundant. I'll push a commit.